### ISSUE TITLE
[UCUM] INVALID_REASON IS NULL and VALID_END_DATE != 2099-12-31

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Vocabulary-v5.0#68]
Just found one UCUM concept with an issue.  

I think you meant to have 2099 not 1999 in the VALID_END_DATE or the concept is actually invalid.

```{sql}
SELECT *
FROM concept
WHERE CONCEPT_ID = 9693
```

```{text}
concept_id	concept_name	valid_start_date valid_end_date
9693	index of reactivity	2014-12-01	1999-12-31
```

**Comment by cgreich** [comment]
Thx. Will fix. My mistake. I put that in manually.

**Comment by ericaVoss** [comment]
Fixed in **v5.0 12-FEB-16**
